ui/src/app/pages/login/account-creation/common.tsx

```diff
@@ -130,18 +130,31 @@ export const Section = (props) => {
   </FormSection>;
 };
 
+export const OptionalSection = (props) => {
+  return <Section subHeader='(Optional)' subHeaderStyle={{fontStyle: 'italic'}} {...props}>
+    {props.children}
+  </Section>;
+};
+
 /**
  * Creates a form section with a dropdown input.
  *
  * @param props
  * @constructor
  */
 export const DropDownSection = (props) => {
-  return <Section header={props.header} subHeader={props.subHeader}>
+  return <Section header={props.header} subHeader={props.subHeader} subHeaderStyle={props.subHeaderStyle}>
```

Comment:
Question Are all dropdownSection in account-creation-survey are using "Optional" tag? If yes can we change DropDownSection to use OptionalSection here itself?

Reply:
I don't want to trust that they are all using optional. I think we'd rather have it extensible and not use OptionalSection